Python/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 install(
-  FILES Python/getkw.py
-  DESTINATION lib/python2.7/site-packages
+  FILES getkw.py
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/python2.7/site-packages
```

Comment:
I think forcing to `python2.7` is wrong. The package might not get found if using Python 3. It's not that big of a problem as of now, I admit. But maybe open an issue on Py3 compatibility so that we don't forget it.

Reply:
I agree. The hard-coding is not nice but the change makes the code better than before.